src/main/java/org/apache/commons/text/similarity/FuzzyScore.java

```diff
@@ -47,7 +72,8 @@
      */
     @Override
     public Integer compare(CharSequence term, CharSequence query) {
-        return compare(term, query, Locale.getDefault());
+        Locale locale = this.locale != null ? this.locale : Locale.getDefault();
```

Comment:
Better make this check in the constructor and always assign a value to the locale field. This way all instance will be in a valid state after construction.